src/libsync/theme.cpp

```diff
@@ -889,4 +889,9 @@ QColor Theme::errorBoxBorderColor() const
     return QColor{"black"};
 }
 
+QPalette Theme::systemPalette() const
+{
+    return QGuiApplication::palette();
```

Comment:
there is a signal emitted when the palette changes, maybe you should connect `QGuiApplication::paletteChanged` to `Theme::systemPaletteChanged` and use it in `Systray` to react to palette changes (like switching between dark to normal color themes)

Reply:
Good idea, done, thanks!